docker/caldera/docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+caldera:
```

Comment:
Don't know what the idea behind these `docker-compose.yml` files in the subdirectories of `docker/` is exactly, but left it here for consistency with the rest (e.g. MQTT/Mongo).